src/app.module.ts

```diff
@@ -7,6 +7,7 @@ import { UserModule } from '@app/user/infrastructure/modules/user.module';
 import jwtConfig from '@config/jwt.config';
 import { APP_GUARD } from '@nestjs/core';
 import { TokenGuard } from '@app/user/infrastructure/guards/token.guard';
+import { UploadsModule } from '@app/uploads/infrastructure/modules/uploads.module';
```

Comment:
```suggestion
import { UploadsModule } from '@app/shared/uploads/infrastructure/modules/uploads.module';
```
اعمل folder سميه shared عشان نحط فيه الmodules اللي راح يستخدمها كل الapplication، زي هادا الmodule